capa/rules/__init__.py

```diff
@@ -940,6 +940,12 @@ def _extract_subscope_rules_rec(self, statement):
             for child in statement.get_children():
                 yield from self._extract_subscope_rules_rec(child)
 
+    def is_internal_rule(self) -> bool:
+        return self.meta.get("namespace", "").startswith("internal/")
+
+    def is_file_limitation_rule(self) -> bool:
+        return self.meta.get("namespace", "") == "internal/limitation/file"
```

Comment:
ah, i guess we had this in `main` because the capa-rules layout concepts aren't otherwise referenced in `rules.py`, so it sort of breaks the abstraction. but, i think its better to keep related code together, rather than have this be totally perfect.

Reply:
ah I see, maybe we should plan to add a "helpers.py" file in capa.rules with such functions in the future?